README.md

```diff
@@ -1,155 +1,152 @@
 
-Alternatively, you could use the custom path type and provide the *customPathToOpenEOS* to manually specificy your installation location. For this method, assume we start at the root of the project and search from there; for example if you installed via the Package Manager remote method, provide something like *"Library/AssetCache/com.robproductions.openeos@someuniqueid"* as shown in your file browser. 
+Alternatively, you can use the custom path type and provide the *customPathToOpenEOS* to manually specify your installation location. For this method, assume the search starts at the root of the project. For example, if you installed via the Package Manager’s remote method, provide something like `"Library/AssetCache/com.robproductions.openeos@someuniqueid"`, as shown in your file browser. 
 
-The installation path parameters are only used in Editor mode so that EOSCore can locate the Plugins folder contained within the EOSSDK folder and dynamically load in EOS DLLs before running the main init steps. This is the recommended approach described in the [C# Getting Started page](https://dev.epicgames.com/docs/epic-online-services/eos-get-started/eossdkc-sharp-getting-started). This allows you to completely shutdown the SDK even in Editor at a later time without having to worry about the lifetime of the Unity Editor itself. As of 1.15, the documentation claims that this is the required method for initializing the SDK properly within the Unity Editor.
+The installation path parameters are used only in **Editor mode**, allowing EOSCore to locate the Plugins folder within the `EOSSDK` folder and dynamically load EOS DLLs before running the main initialization steps. This method is recommended on the [C# Getting Started page](https://dev.epicgames.com/docs/epic-online-services/eos-get-started/eossdkc-sharp-getting-started) because it allows you to shut down the SDK in Editor mode without being constrained by the Unity Editor’s lifecycle. As of version 1.15, this is the required method for properly initializing the SDK in Unity Editor.
 
-With just those 2-3 parameters as input into **EOSCore.Init()**, you can quickly start up the SDK and retrieve the **PlatformInterface** that you should use in all future requests to the SDK. As long as the returned platform wasn't null, you have successfully started up the SDK. Here's an example:
+With just 2-3 parameters as input into **EOSCore.Init()**, you can quickly start the SDK and retrieve the **PlatformInterface** for use in all future SDK requests. If the returned platform is not `null`, you have successfully started the SDK. Here’s an example:
 
 ```cs
 var initSet = new EOSInitSet()
 {
-	productName = "Test Game",
```

Comment:
I believe the code uses enforced tabs, so would prefer to leave the example the same too